manifests/params.pp

```diff
@@ -36,6 +36,11 @@
   $rdeck_home = '/var/lib/rundeck'
   $service_logs_dir = '/var/log/rundeck'
 
+  $rdeck_uuid = $::serialnumber ? {
+    undef   => fqdn_uuid($::fqdn),
+    default => $::serialnumber,
```

Comment:
`$::serialnumber` is highly variable. It's taken from information in `/sys/class/dmi/id/product_name`, which can vary alot between something hard-coded for cloud instances or differing for bare metal. 

Technically this will fix an issue currently affecting any machine with this fact missing:

> When the 'serialnumber' fact is not present the 'rundeck.server.uuid'
setting is set to 'undef'. Also when the 'strict_variables' setting
is set to true, the Puppet parser will fail if the 'serialnumber' fact
is not present.